eos/b-decays/b-to-3l-nu.cc

```diff
@@ -88,7 +88,7 @@ namespace eos
             }
             if (opt_l.str() == opt_lprime.str())
             {
-                throw InvalidOptionValueError("opt_lprime", opt_l.str(), "e, mu if it is not the value of l_prime");
+                throw InvalidOptionValueError("opt_lprime", opt_l.str(), "e, mu, tau if it is not the value of l_prime");
```

Comment:
Please split this into a separate commit ``Add means to select l=tau in B->3lnu observables``.